harmony_netcdf_to_zarr/mosaic_utilities.py

```diff
@@ -243,6 +248,7 @@ def _aggregate_output_dimensions(self):
                 self.output_dimensions[dimension_name] = self._get_output_dimension(
                     dimension_name, all_input_values, output_dimension_units
                 )
+                """
```

Comment:
Normally I would remove commented code, but this will likely be reimplemented when spatial aggregation is required.